crates/agentgateway/src/types/local.rs

```diff
@@ -1093,8 +1124,81 @@ async fn split_frontend_policies(
 	if let Some(p) = access_log {
 		add(FrontendPolicy::AccessLog(p), "accessLog");
 	}
-	if let Some(p) = tracing {
-		add(FrontendPolicy::Tracing(p), "tracing");
+	if let Some(local_config) = tracing {
+		// Parse CEL attributes
+		let attributes = local_config
+			.attributes
+			.iter()
+			.map(|a| {
+				let expr = cel::Expression::new(&a.value)
+					.map_err(|e| anyhow::anyhow!("invalid CEL in tracing attribute: {}", e))?;
+				Ok(types::agent::TracingAttribute {
+					name: a.name.clone(),
+					value: Arc::new(expr),
+				})
+			})
+			.collect::<anyhow::Result<Vec<_>>>()?;
+
+		// Extract endpoint directly from the local backend config
+		let endpoint = match &local_config.provider_backend {
+			SimpleLocalBackend::Service { name, port } => {
+				format!("http://{}:{}", name.hostname, port)
+			},
+			SimpleLocalBackend::Opaque(target) => {
+				// Target is already in "host:port" format
+				format!("http://{}", target)
```

Comment:
It looks like we're hard coding the target to be `http`. If the OTLP endpoint requires TLS, this will silently break it. Let's add a TODO to support tls. (Maybe add a insecure/secure field or take the URI input)